### DISCUSSION TITLE
Backups

### DISCUSSION THREAD

**Discussion by lyk4ndna** [mbround18/valheim-docker#439] [Help!] [answered]
I see on the main page: 
Variable | Default | Required | Description
AUTO_BACKUP | 0 | FALSE | Set to 1 to enable auto backups.
Backups are stored under **/home/steam/backups** which means you will have to add a volume mount for this directory.

I don't have a folder named steam when I click home, just valheim (I fully admit I might have set it up wrong when I first initialized the container).

When I click the valheim folder it shows me the docker-compose.yml, saves, logs, another valheim folder with a lock on it (contains the worlds, saves, steamclient, etc).

The backups are stored under /home/steam/backups....I feel like it should be obvious to me, but it's not...do I just create a new folder in home named steam and another folder in steam as backups?

What if I want to put the backups onto a USB drive?

**Comment by mbround18** [comment, marked as answer]
hmmm, you can mount the USB drive to the valheim folder, whats your docker compose look like?

**Reply by lyk4ndna** [reply]
I’m a dumb dumb…totally didn’t have the backup volume on the docket compose yml. Just added it. Will update in about 30 minutes if it went through.

**Reply by lyk4ndna** [reply]
Fixed the docker compose file to include backup and the backup volume, now it's working correctly. Easy fix. :)
